erpnext/hr/doctype/driver/driver.json

```diff
@@ -159,13 +162,23 @@
    "role": "HR Manager",
    "share": 1,
    "write": 1
+  },
+  {
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "report": 1,
+   "role": "Sales User",
```

Comment:
Add "Select" permission for "Sales User (Read Only)" as well

Reply:
done